logic-state-indexer/src/lib.rs

```diff
@@ -214,30 +220,37 @@ pub async fn handle_streamer_message(
         near_client,
         db_manager,
     );
-    let handle_block_future = db_manager.save_block_with_chunks(
-        block_height,
-        block_hash,
-        streamer_message
-            .block
-            .chunks
-            .iter()
-            .map(|chunk| {
-                (
-                    chunk.chunk_hash.to_string(),
-                    chunk.shard_id,
-                    chunk.height_included,
-                )
-            })
-            .collect(),
-    );
-    let handle_state_change_future = handle_state_changes(
-        &streamer_message,
-        db_manager,
-        block_height,
-        block_hash,
-        &indexer_config,
-        shard_layout,
-    );
+    let handle_block_future = Retry::spawn(retry_strategy.clone(), || async {
```

Comment:
Is there any way to emit logs to see that we've failed to save and will retry? Ideally, it should have a "recovery" log message if the retries count is greater than 1.

Reply:
Unfortunatelly `tokio_retry` didn't have a logic for tracking number of retries, so I decided to remove this dependency (technically I could introduce a mutex for this, but it's not worth it)